src/test/platform/activation.cpp

```diff
@@ -1,29 +1,29 @@
 #include "pch.h"
 
-// TODO: enable xplat activation test
-#if XLANG_PLATFORM_WINDOWS
-#include "winrt_helpers.h"
-
 TEST_CASE("Simple activation")
 {
     xlang_error_info* result{};
 
     // TODO: Update cpp projection to emit char16_t names
-    std::u16string_view class_name{ u"Component.Edge.ZeroClass" };
+    std::u16string_view class_name{ u"AbiComponent.Widget" };
     xlang_string_header str_header{};
     xlang_string str{};
 
     result = xlang_create_string_reference_utf16(class_name.data(), static_cast<uint32_t>(class_name.size()), &str_header, &str);
     REQUIRE(result == nullptr);
 
-    winrt::Windows::Foundation::IUnknown factory{};
-    xlang_guid iid = to_guid(winrt::guid_of<winrt::Windows::Foundation::IActivationFactory>());
-    result = xlang_get_activation_factory(str, iid, winrt::put_abi(factory));
+    xlang_unknown* factory{};
```

Comment:
>xlang_unknown [](start = 4, length = 13)

com_ptr (or something like it, which shouldn't require XLANG_PLATFORM_WINDOWS)

Reply:
Agree 100%...when we have a xplat implementation of com_ptr.

Reply:
added #185 to track